src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -719,6 +719,11 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
                 _cursorTimer = std::nullopt;
             }
 
+            // Set up a hover timer, but don't start it yet
+            int hoverTime = 500;
+            _hoverTimer.Interval(std::chrono::milliseconds(hoverTime));
+            _hoverTimer.Tick({ get_weak(), &TermControl::_HoverTimerTick });

```

Comment:
wait do we ever start the timer?

Reply:
Its commented out for now (explanation in previous reply)